template.cpp

```diff
@@ -1,35 +0,0 @@
-
-
-
-#include <bits/stdc++.h>
-using namespace std;
-
-typedef long long ll;
-typedef vector<int> vi;
-typedef vector<long long> vll;
-typedef pair<int,int> pi;
-
-#define F first
-#define S second
-#define PB push_back
-#define MP make_pair
-#define FOR(i, N) for(int i =0; i<N; i++) 
-
-void solve()
-{
-  cout<<"sol";
-}
-
-int main() {
-	ios::sync_with_stdio(0);
-    cin.tie(0);
-	int t = 1;
-	// cin >> t;
-	while(t--) {
-		solve();
-		cout<<"\n";
-	}
-
-	return 0;
-}
-
```

Comment:
#include <bits/stdc++.h>
using namespace std;

typedef long long ll;
typedef vector<int> vi;
typedef vector<long long> vll;
typedef pair<int,int> pi;

#define F first
#define S second
#define PB push_back
#define MP make_pair
#define FOR(i, N) for(int i =0; i<N; i++) 

void solve()
{
  cout<<"sol";
}

int main() {
	ios::sync_with_stdio(0);
    cin.tie(0);
	int t = 1;
	// cin >> t;
	while(t--) {
		solve();
		cout<<"\n";
	}

	return 0;
}